Dockerfile generation

shelmich opened the pull request:
Added functions to aid in the generation of dockefiles

Commit:
fixing

Commit:
fixed

Commit:
fixed conflicts

Merge remote-tracking branch 'origin/master' into dockerfile-generation

# Conflicts:
#	DESCRIPTION
#	NAMESPACE

Commit:
Added Dockerfile to .gitignore, added basic_dockerfile

Commit:
file cleanup, started base dockerfile class

Commit:
added write_dockerfile functionality

Commit:
changed return objectg for dockerfile()

Commit:
Added function to insert a blank row

Commit:
fixed issue of 0-length variable name

benjamincrary committed:
Added set_from

benjamincrary committed:
Merge branch 'dockerfile-generation' of https://github.com/uncoast-unconf/whaler into dockerfile-generation

benjamincrary committed:
Added set_from

Commit:
added functionality for setting environment variables, and fixed bugs

Commit:
added insert_blank at the end of each set_from call

Commit:
Added but did not complete set_label, bug fixes

benjamincrary committed:
Added check_for_from

Commit:
added set_run function

Commit:
finished set_label

benjamincrary committed:
Added check_for_from

Merge branch 'dockerfile-generation' of https://github.com/uncoast-unconf/whaler into dockerfile-generation

# Conflicts:
#	NAMESPACE
#	R/set_from.R
#	man/insert_blank.Rd
#	man/set_from.Rd

benjamincrary committed:
Merge branch 'dockerfile-generation' of https://github.com/uncoast-unconf/whaler into dockerfile-generation

ellisvalentiner commented on R/set_env.R:
```diff
@@ -0,0 +1,30 @@
+#' Title
```
Can you change `Title` to something more informative?

Commit:
fixed bug in set_from where FROM was duplicated in name and element

Commit:
Fixed bug in set_from where FROM was being duplicated

Merge branch 'dockerfile-generation' of https://github.com/uncoast-unconf/whaler into dockerfile-generation

# Conflicts:
#	R/set_from.R

Commit:
add readme and pkgdown site

Commit:
Added informative title

Commit:
fixed conflicts

Merge branch 'master' into dockerfile-generation

# Conflicts:
#	NAMESPACE

ellisvalentiner commented:
Fixes #4
